app/components/search_bar/search_box.js

```diff
@@ -339,45 +339,46 @@ export default class Search extends Component {
                 ]}
                 onLayout={this.onLayout}
             >
-                <AnimatedTextInput
-                    ref='input_keyword'
-                    style={[
-                        styles.input,
-                        this.props.placeholderTextColor && {color: this.props.placeholderTextColor},
-                        this.props.inputStyle && this.props.inputStyle,
-                        this.props.inputHeight && {height: this.props.inputHeight},
-                        this.props.inputBorderRadius && {borderRadius: this.props.inputBorderRadius},
-                        {
-                            width: this.inputFocusWidthAnimated,
-                            paddingLeft: this.inputFocusPlaceholderAnimated,
-                        },
-                        this.props.shadowVisible && {
-                            shadowOffset: {width: this.props.shadowOffsetWidth, height: this.shadowHeight},
-                            shadowColor: this.props.shadowColor,
-                            shadowOpacity: this.shadowOpacityAnimated,
-                            shadowRadius: this.props.shadowRadius,
-                        },
+                <View style={this.props.inputStyle}>
```

Comment:
Should I change this prop name? i am 0/5 in this so left it this way. Lemme know if anyone prefers something like `inputWrapperStyle`

Reply:
Yeah that change sounds good, also what about android?

Reply:
There is no highlight for text so no changes are required for android.

Reply:
Made changes to renaming prop